eng/testing/linker/trimmingTests.targets

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <TestConsoleAppSourceFiles Include="$(MSBuildProjectDirectory)\*.cs" />
+    <TestConsoleAppSourceFiles Condition="'@(TestConsoleAppSourceFiles)' == ''" Include="$(MSBuildProjectDirectory)\*.cs" />
```

Comment:
not sure why this shows as a diff since this is the state this line is in master today...

Reply:
Hmm I wasn't synced to master when I manually made this change. Hopefully, the diff won't cause any adverse effects.

Reply:
Fixed after moving my changes on top of master.